### ISSUE TITLE
12 bit color depth 

### ISSUE THREAD

**Issue by Uj947nXmRqV2nRaWshKtHzTvckUUpD** [VirtualDisplay/Virtual-Display-Driver#18]
Hi. Thank you for this amazing fork. I am finally able to play games with HDR in a hyper-v vm with susnhine/moonlight. my question is why the code sets 10 bit instead of 12 bit color depth. I still see some rare slight color banding at times in cyberpunk in dark areas

**Comment by zjoasan** [comment]
After some research I've found out that yes it's now possible to have the VDD in HDR+(eg. 12 bit), will check the rest of the dev-team about implementing it. I gather that it might need a few changes, but I can't predict if it will effect the majority of users that wants HDR10.

Or if it's as simple as HDR+ edid, then it will be possible as soon as the companion is ready.

**Comment by itsmikethetech** [comment]
HDR bit-based color depth changes are now supported in the beta.

**Comment by bud3699** [comment]
Open xml. Head over to hdr+ and switch that bad boy on

**Comment by itsmikethetech** [comment]
I believe that's the encoded setting. Could Sunshine be setting AV1 encoding to 10-bit? I might take a look at their code later to see if it's a setting that can be updated. Will take some testing to figure out what's actually being sent/recieved.

**Comment by itsmikethetech** [comment]
Makes sense. Yes, it does look like this would need to be integrated on Sunshines side. It may have already been. The driver now contains the working functionality for this feature, so I'll close, but feel free to open if any other questions come up.
